### PR TITLE
Jwoo/ios toggle accessibility

### PR DESCRIPTION
**Pull request by jwoo-msft**
## Related Issue
ADO https://microsoft.visualstudio.com/OS/_queries/edit/29559531/?triage=true

## Description
MAS1 4 11_ColumnSet Input Toggle VerticalStretch

## Sample Card
https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.1/Tests/ColumnSet.Input.Toggle.VerticalStretch.json
image

## How Verified
1. Verified using Colour Contrast Analyzer.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5090)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib [hunk 1]
```diff
@@ -14,10 +12,7 @@
         <datePicker contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" datePickerMode="dateAndTime" minuteInterval="1" id="j3j-us-bMu">
             <rect key="frame" x="0.0" y="0.0" width="375" height="215"/>
             <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMaxY="YES"/>
-            <color key="backgroundColor" white="0.66666666666666663" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-            <date key="date" timeIntervalSinceReferenceDate="544572879.81824601">
-                <!--2018-04-04 22:14:39 +0000-->
-            </date>
```
Comment (RebeccaAnne) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib:20]: Out of curiosity, what was this date about?

**Review by RebeccaAnne — APPROVED** [review]
:shipit:

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib (hunk 1 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib:20]: I'm not quite sure, either. Let me check on it.

Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib]: the change is from a topic branch this branch was based on.

Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib]: as i clean up the commits for the merge, i reverted those changes.

**Comment by jwoo-msft** [comment]
FYI, Android CI was re-run after the cancelation, and confirmed that CI complets.

**Pull request merged**